junseoungbin_new/src/main/java/com/likelion/junseoungbin_new/oauth/api/AuthLoginController.java

```diff
@@ -0,0 +1,22 @@
+package com.likelion.junseoungbin_new.oauth.api;
+
+import com.likelion.junseoungbin_new.oauth.api.dto.Token;
+import com.likelion.junseoungbin_new.oauth.application.AuthLoginService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/login/oauth2")
+public class AuthLoginController {
+    private final AuthLoginService authLoginService;
+    @GetMapping("/code/google")
+    public Token googleCallback(@RequestParam(name = "code") String code){
+        String googleAccessToken = authLoginService.getGoogleAccessToken(code);
+        return loginOrSignup(googleAccessToken);
+    }
```

Comment:
클래스 내 줄바꿈 규칙에 대해서 의식하시면서 코드 작성하시면 좋을 것 같아요!
Java 코딩 컨벤션 관련 블로그를 첨부해두겠습니다~
[Java 코딩 컨벤션](https://naver.github.io/hackday-conventions-java/)

```suggestion
    private final AuthLoginService authLoginService;
    
    @GetMapping("/code/google")
    public Token googleCallback(@RequestParam(name = "code") String code){
        String googleAccessToken = authLoginService.getGoogleAccessToken(code);
        return loginOrSignup(googleAccessToken);
    }
```

Reply:
넵!